rm-datasource/src/test/java/io/seata/rm/datasource/exec/MySQLInsertExecutorTest.java

```diff
@@ -565,6 +579,20 @@ public void test_checkPkValues() {
         Assertions.assertFalse(insertExecutor.checkPkValuesForSinglePk(pkValues, false));
     }
 
+    @Test
+    public void test_autoGeneratePks() throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+        Method method = MySQLInsertExecutor.class.getDeclaredMethod("autoGeneratePks", new Class[]{Long.class, String.class, Integer.class});
+        method.setAccessible(true);
```

Comment:
Mark the target method as package-private would be better, since reflect is slower and easy to go fault.

Reply:
reflect only be used in test case